src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -23,7 +23,7 @@
 
 STORAGE_RESOURCE_ENDPOINT = "https://storage.azure.com"
 SERVICES = {'blob', 'file'}
-AZCOPY_VERSION = '10.3.3'
```

Comment:
Do we need this PR? https://github.com/Azure/azure-cli/pull/14353

Reply:
to prevent conflict.